addon/components/sequential-render.js

```diff
@@ -35,7 +34,6 @@
   @class sequential-render
   @public
   @yield {Hash} hash
-  @yield {Any} hash.content The response from performing fetchDataTask.
```

Comment:
Shouldn't we rename this to getData?

@yield {Any} hash.content The response from yielding getData function.

Reply:
yes , have accommodated in latest commit